var/spack/repos/builtin/packages/minighost/package.py

```diff
@@ -55,8 +55,7 @@ def build_targets(self):
             targets.append('PROTOCOL=-D_MG_MPI')
             targets.append('FC={0}'.format(self.spec['mpi'].mpif77))
             targets.append('CC={0}'.format(self.spec['mpi'].mpicc))
-            targets.append(
-                'LIBS=-lm -lmpi_cxx')
+            targets.append('LIBS=-lm -lmpi_cxx -lgfortran')
```

Comment:
Is this needed for compilers that are not gcc?

Reply:
Most likely not...